packages/tools/pie-wrapper-react/scripts/add-react-wrapper.js

```diff
@@ -28,14 +31,12 @@ export function addReactWrapper (customElementsObject, folderName = process.argv
             sortedModules = value.sort((a, b) => a.path.length - b.path.length);
 
             value.forEach((k) => {
-                if (k.path.includes(folderName)) {
-                    k.declarations.forEach((decl) => {
-                        if (decl.customElement === true) {
-                            const componentSelector = k.declarations.find((i) => i.kind === 'variable' && i.name === 'componentSelector');
-                            components.push({ class: { ...decl, tagName: componentSelector?.default.replace(/'/g, '') ?? decl.tagName }, path: k.path.replace('index.js', 'react.ts') });
-                        }
-                    });
-                }
+                k.declarations.forEach((decl) => {
+                    if (decl.customElement === true) {
+                        const componentSelector = k.declarations.find((i) => i.kind === 'variable' && i.name === 'componentSelector');
+                        components.push({ class: { ...decl, tagName: componentSelector?.default.replace(/'/g, '') ?? decl.tagName }, path: `${folderName}/react.ts` });
+                    }
```

Comment:
It would be good to compare a newer manifest to a section of the previous one and see if anything has changed (more of manual check than a written test)

Reply:
Discussed with @jamieomaguire, and this is all covered by existing tests :)